content/en/docs/tasks/administer-cluster/kubeadm/adding-linux-nodes.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Adding Linux worker nodes
+content_type: task
+weight: 50
+---
+
+<!-- overview -->
+
+This page explains how to add Linux worker nodes to a kubeadm cluster.
+
+## {{% heading "prerequisites" %}}
+
+* Each joining worker node has installed the required components from
+[Installing kubeadm](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/), such as,
+kubeadm, the kubelet and a {{< glossary_tooltip term_id="container-runtime" text="container runtime" >}}.
+* A running kubeadm cluster created by `kubeadm init` and following the steps
+in the document [Creating a cluster with kubeadm](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/).
+
+<!-- steps -->
+
+## Adding Linux worker nodes
+
+To add new Linux worker nodes to your cluster do the following for each machine:
+
+1. Connect to the machine by using SSH or another method.
+1. Become root (e.g. `sudo su -`).
+1. Run the command that was output by `kubeadm init`. For example:
+
+  ```bash
+  kubeadm join --token <token> <control-plane-host>:<control-plane-port> --discovery-token-ca-cert-hash sha256:<hash>
+  ```
+
+### Additional information for kubeadm join
+
+{{< note >}}
+To specify an IPv6 tuple for `<control-plane-host>:<control-plane-port>`, IPv6 address must be enclosed in square brackets, for example: `[2001:db8::101]:2073`.
+{{< /note >}}
+
+If you do not have the token, you can get it by running the following command on the control plane node:
+
+```bash
+kubeadm token list
+```
+
+The output is similar to this:
+
+```console
+TOKEN                    TTL  EXPIRES              USAGES           DESCRIPTION            EXTRA GROUPS
+8ewj1p.9r9hcjoqgajrj4gi  23h  2018-06-12T02:51:28Z authentication,  The default bootstrap  system:
+                                                   signing          token generated by     bootstrappers:
+                                                                    'kubeadm init'.        kubeadm:
+                                                                                           default-node-token
+```
+
+By default, tokens expire after 24 hours. If you are joining a node to the cluster after the current token has expired. You can create a new token by running the following command on the control plane node:
+
+```bash
+kubeadm token create
```

Comment:
```suggestion
# Run this on a control plane node
sudo kubeadm token create
```

Reply:
updated, minus `# Run this on a control plane node`. the prev sentence already mentions `...running the following command on the control plane node`